samcli/local/lambdafn/remote_files.py

```diff
@@ -0,0 +1,53 @@
+"""
+Helper methods to handle files in remote locations.
+"""
+
+import os
+import logging
+from pathlib import Path
+
+import requests
+
+from samcli.lib.utils.progressbar import progressbar
+from samcli.local.lambdafn.zip import unzip
+
+LOG = logging.getLogger(__name__)
+
+
+def unzip_from_uri(uri, layer_zip_path, unzip_output_dir, progressbar_label):
```

Comment:
was this just moved from the other file below or did something change?

Reply:
yes, refactored it to a separate module.